public/js/controllers/data_controller.js

```diff
@@ -74,6 +74,31 @@ export class DataController {
         }, error => console.log(error));    
     }
 
+    static watchPipelinesMetrics(onChange) {
+        const TIMERANGE = 31;
+        
+        let offset = new Date(), 
+            offsetString, 
+            iso = d3.utcFormat("%Y-%m-%dT%H:%M:%S+%L");
+
+        offset.setDate(offset.getDate() - TIMERANGE);
+        offsetString = iso(offset);
+
+        let pipelineMetrics = [];
+        return mediadb.collection("metrics/pipelines/pipeline_logs")
+            .where("timestamp", ">", offsetString)
```

Comment:
(If we're concerned strictly about the table, I don't think this is the most efficient means to retrieve data, as we're only interested in the latest documents. However, given that @DanielMwendwa you mentioned the view that shows older projects too is progressing well, no need to change here)

Reply:
Noted, Thanks.